src/commands/init.ts

```diff
@@ -187,19 +193,55 @@ export default class Init extends Command {
     }
     packageJson.scripts = { ...packageJson.scripts, ...scripts };
 
-    if (initializationType === InitType.freshExpress) {
-      packageJson.devDependencies = { ...packageJson.devDependencies, ...packageJsonData.devDependencies };
-    }
+    packageJson.dependencies = await this.addDependencies(packageJsonData.dependencies, packageJson.dependencies, flags.projectDir);
+    packageJson.devDependencies = await this.addDependencies(packageJsonData.devDependencies, packageJson.devDependencies, flags.projectDir);
 
     fs.writeFileSync(path.resolve(flags.projectDir, 'package.json'), JSON.stringify(packageJson, null, 2));
+  }
+
+  private async addDependencies(
+    dependencies: string[],
+    toBeAdded: { [key: string]: string },
+    projectDir: string
+  ): Promise<{ [key: string]: string }> {
+    const copy: { [key: string]: string } = {};
+    if (toBeAdded) {
+      Object.entries(toBeAdded).forEach(([key, value]) => (copy[key] = value));
+    }
+    const promises = dependencies.map(async dependency => {
+      const latestVersion = await this.getVersionOfDependency(dependency, projectDir);
+      if (latestVersion) {
+        copy[dependency] = `^${latestVersion}`;
+      }
+      return Promise.resolve();
+    });
+    await Promise.all(promises);
+    return copy;
+  }
 
+  private async getVersionOfDependency(dependency: string, projectDir: string): Promise<string | undefined> {
     try {
-      await execa('npm', ['install', '@sap/cloud-sdk-core'], {
-        cwd: flags.projectDir
-      });
-      await execa('npm', ['install', '--save-dev', '@sap/cloud-sdk-test-util'], { cwd: flags.projectDir });
-    } catch (error) {
-      this.error(error, { exit: 12 });
+      return (await execa('npm', ['view', dependency, 'version'], { cwd: projectDir })).stdout;
+    } catch (err) {
+      this.error(`Error in finding version for dependency ${dependency}`);
+      return undefined;
+    }
+  }
+
+  private async installDependencies(flags: Flags, initType: InitType) {
+    try {
+      switch (initType) {
```

Comment:
We can probably just get rid of this switch and just always install.

Reply:
ok like it. keep it simple